### PR TITLE
index.md

### PR DESCRIPTION
**Pull request by serg-t**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/russian/php/PHP Variables/variables_php.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
```
Comment (Asjas) [inline guide/russian/php/PHP Variables/variables_php.md]: Hi @serg-t 

Please change the folder name to be lowercase and to be separated with hyphens. Also change the filename back to `index.md`.

Then also use `code blocks` for any code examples you use in the article.

**Review by Asjas — DISMISSED** [review]

**Comment by RandellDawson** [comment]
@greggubarev Can you review this one?

**Review by greggubarev — CHANGES_REQUESTED** [review]
@serg-t please add example from 8 line, and add markdown for highlighting code

**Review by greggubarev — APPROVED** [review]

**Review by greggubarev — APPROVED** [review]
LGTM 👍 :dart:

Thank you for the PR ✨ 🎉
 We're happy to accept these changes and look forward to your future contributions :clap:

Happy coding! :octocat:

**Pull request merged**